.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python

```

Comment:
I haven't used travis before. Does this push to docker hub or does travis have it's own private registry used for testing?

Reply:
you have to set your environment variables you see referenced in the travis UI itself with your docker hub credentials. It does not have a local registry. This builds the container and pushes it, then tests it by pulling and running it.